site/site.go

```diff
@@ -290,8 +291,9 @@ func (h *Handler) renderHTMLWithState(rw http.ResponseWriter, r *http.Request, f
 	// Cookies are sent when requesting HTML, so we can get the user
 	// and pre-populate the state for the frontend to reduce requests.
 	apiKey, actor, _ := httpmw.ExtractAPIKey(rw, r, httpmw.ExtractAPIKeyConfig{
-		Optional: true,
-		DB:       h.opts.Database,
+		Optional:      true,
+		DB:            h.opts.Database,
+		OAuth2Configs: h.opts.OAuth2Configs,
```

Comment:
Should we pass other extract api key configs, like `DisableSessionExpiryRefresh`, too?

Reply:
/ping @deansheather (ref: https://github.com/coder/coder/pull/5976)

Reply:
Yeah we should be setting that flag everywhere we use it, so this is definitely a bug.

For this specific case though, we can just set it to `true` because it doesn't matter if loading the frontend doesn't refresh the token (because the frontend will make API requests after loading the page).

Reply:
Thanks @deansheather, added the always true and an accompanying comment.